python/examples/embedding/wasi-py-rs/src/bin/py-func-caller.rs

```diff
@@ -0,0 +1,35 @@
+use pyo3::{append_to_inittab, PyResult};
+
+use wasi_py_rs::call_function;
+use wasi_py_rs::py_module::make_person_module;
+
+pub fn main() -> PyResult<()> {
+    append_to_inittab!(make_person_module);
+
+    let function_code = "def my_func(*args, **kwargs):
```

Comment:
Just a minor suggestion. For other language scripts, I usually define them in a separate `*.py` file and add them via the `include_str!` macro. With this approach, people can check the Python code using the proper colors, so it's easier to follow.

However, this code is quite short so feel free to leave it inline.